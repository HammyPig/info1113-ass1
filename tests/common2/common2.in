common 100000 100001
add mEmBer John Smith
common 100000 100001
add Member Amy Adams
common 100000 100001
add Collection books.csv
common 100000 100000
common 100000 100001
common 100000 100001 100002
add member peterpoo
common 100000 100001 100002