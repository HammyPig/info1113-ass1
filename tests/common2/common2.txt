can recognise if there are no members/books, or if members are invalid