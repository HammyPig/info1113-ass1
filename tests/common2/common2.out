user: No members in system.

user: Success.

user: No books in system.

user: Success.

user: No books in system.

user: 17 books successfully added.

user: Duplicate members provided.

user: No common books.

user: No such member in system.

user: Success.

user: No common books.

user: Ending Library process.
